.github/workflows/test-installers.yml

```diff
@@ -6,6 +6,10 @@ permissions:
 on:
   workflow_dispatch:
     inputs:
+      version:
+        type: string
+        description: 'Version of the installer to download'
+        required: true
```

Comment:
Will it work with a required version? Do we know the version for beta and stable, for example? Wasn't that only really used for the dev version?

Reply:
😬 you're probably right. This would happen with a run off `main` or one of the release branches, right?

Reply:
If I remember correctly it is optional so that you specify it for dev, but then for beta/stable it gets read out of package.json. Probably indirectly in hadron build info somewhere.